security/tls-security-profiles.adoc

```diff
@@ -5,10 +5,28 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-TODO
+////
+To do list:
+* Add release note for this whole feature
+* Add known issue to release note for the doc for kubelet not listing the right ciphers
+////
 
-// TODO: The overview
-// include::modules/todo.adoc[leveloffset=+1]
+Cluster administrators can choose which cipher suites to use for their {product-title} cluster. A TLS security profile defines the TLS ciphers that various {product-title} components must use when connecting with other components through the cluster. This provides a convenient way to ensure that the {product-title} components use cryptographic libraries that do not allow known insecure protocols, ciphers, or algorithms.
```

Comment:
The TLS profiles are a means for servers to regulate which ciphers a connecting client may use when connecting to these servers. The description here feels as if it was making an assumption that the TLS profiles are a client-side configuration.